config.js

```diff
@@ -26,14 +26,14 @@ const config = {
     //       </div>
     //     </a>
     //   </li>`,
-    links: [{ text: '', link: '' }],
-    // search: {
-    //   enabled: true,
-    //   indexName: '',
-    //   algoliaAppId: process.env.GATSBY_ALGOLIA_APP_ID,
-    //   algoliaSearchKey: process.env.GATSBY_ALGOLIA_SEARCH_KEY,
-    //   algoliaAdminKey: process.env.ALGOLIA_ADMIN_KEY,
-    // },
+    links: [{ text: 'CONTACT US', link: 'mailto:mait@plutovr.com' }],
+    search: {
+      enabled: true,
+      indexName: 'dev_MAIT',
```

Comment:
One more non-blocking question: is that index going to change in different environments? That's my reading of the `dev_` prefix, absent any other context.